operator/api/redpanda/v1alpha2/testdata/crd-docs.adoc

```diff
@@ -440,13 +502,223 @@ _Underlying type:_ _ClusterConfiguration_
 
 
 
+
+
 .Appears In:
 ****
 - xref:{anchor_prefix}-github-com-redpanda-data-redpanda-operator-operator-api-redpanda-v1alpha2-config[$$Config$$]
 ****
 
 
 
+
+
+[id="{anchor_prefix}-github-com-redpanda-data-redpanda-operator-operator-api-redpanda-v1alpha2-clusterconfigurationappliedcondition"]
+==== ClusterConfigurationAppliedCondition
+
+_Underlying type:_ _string_
+
+"ConfigurationApplied" - This condition indicates whether cluster
+configuration parameters have currently been applied to a cluster for the
+given generation.
+
+
+This condition defaults to "False" with a reason of "NotReconciled" and must
+be set by a controller when it subsequently reconciles a cluster.
+
+
+
+[cols="20a,80a", options="header"]
+|===
+| Reason | Description |
+| `Applied` | This reason is used with the "ConfigurationApplied" condition when it +
+evaluates to True because a cluster has had its cluster configuration +
+parameters applied. +
+ |
+| `NotApplied` | This reason is used with the "ConfigurationApplied" condition when it +
+evaluates to False due to some implementation-specific condition, such as +
+when no brokers have been created and thus we can't attempt a configuration. +
+ |
+| `Error` | This reason is used when a cluster has only been partially reconciled and we +
+have early returned due to a retryable error occurring prior to applying the +
+desired cluster state. If it is set on any non-final condition, then the +
+condition "Quiesced" will be False with a reason of "SillReconciling". +
```

Comment:
There are a few instances of this typo. `SillReconciling` should be `StillReconciling`